NutritionManagement/src/calculation/NutritionCalculator.java

```diff
@@ -5,6 +5,13 @@
 
 public class NutritionCalculator {
 
+    /**
+     * (2000, 50, 275, 78)
+     * as in
+     * https://www.fda.gov/food/nutrition-facts-label/daily-value-nutrition-and-supplement-facts-labels
+     */
+    private static NutritionValue referenceNutritionValue = new NutritionValue(2000, 50, 275, 78);
```

Comment:
It's good to see constants being used for reference values., ensures consistency and maintainability.